.gitignore

```diff
@@ -17,3 +17,6 @@ sway-lib-std/Forc.lock
 
 # Forc's build directory
 out
+
+# Files specific to IDEs
+.vscode
```

Comment:
Platform- or IDE-specific excludes should be in `.git/info/exclude` or your personal `~/.gitignore`.